src/components/ResetPasswordRequest.tsx

```diff
@@ -0,0 +1,65 @@
+// ResetPasswordRequest component
+import React, { useState } from 'react';
+import axios from 'axios';
+import Alert from './Alert';
+import logo from "../assets/logo.png";
+
+const VITE_BACKEND_URL = 'https://dev-kosj-api.fly.dev'; // Your backend URL
+
+const RequestResetPassword = () => {
+  const [email, setEmail] = useState('');
+  const [alertMessage, setAlertMessage] = useState('');
+  const [showAlert, setShowAlert] = useState(false);
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    try {
+      await axios.post(`${VITE_BACKEND_URL}/password/forgot`, { email });
```

Comment:
Please check other examples of how we do requests. 
- First we create a service (in this case, add the functions to auth service file)
- And use our axios wrapper, not the one directly imported, because the authentication won't work like this (for this request it would be fine because it doesnt require auth, but still, not a good practise to use different methods)
- Lastly, Never use hardcoded variables like the VITE_BACKEND_URL. They come from .env and we already have the boilerplate for requests

Reply:
Also, don't use await, better to use the .then and .catch callbacks, like we do it in other places.

Reply:
Something like
```suggestion
      authService.forgotPassword(email)
         .then(() => some code, set alert, set alert true)
         .catch(() => other code)
```

And remove the try catch

Reply:
Fixed it